cardano-cli/src/Cardano/CLI/Byron/Key.hs

```diff
@@ -1,13 +1,7 @@
-{-# LANGUAGE CPP #-}
 {-# LANGUAGE GeneralisedNewtypeDeriving #-}
-{-# LANGUAGE LambdaCase #-}
-{-# LANGUAGE NoMonomorphismRestriction #-}
 
-{-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
 {-# OPTIONS_GHC -Wno-missing-local-signatures #-}
```

Comment:
That would be a good one to remove.

Reply:
We have to fix the monolithic error types first :-)